### PR TITLE
added an article that explains the Office 365 connection

### PR DESCRIPTION
**Pull request by vinkobedek**
added an article that explains the Office 365 connection

### CODE REVIEW

File: installation-and-configuration/office-365-connection-details.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+When configuring SysKit Trace an Azure Active Directory Application will be created and pre-consented for the user to collect the data from the tenant.
+You can find the SysKit Trace application in the [Azure Portal](https://portal.azure.com/#blade/Microsoft_AAD_IAM/ActiveDirectoryMenuBlade/RegisteredApps) once SysKit Trace has been configured. 
+
+There are two ways to authenticate an Azure Active Directory Application with Azure AD when using the applications identity.
+1. application secrets
+2. certificates  
+
+SysKit Trace uses certificates because most of the workloads that are collected support it.
+A certificate is created and imported into the current user personal certificates for both the user configuring SysKit Trace and the service account that is configured to run the SysKit Trace service.
+This is the reason why we also require a specific windows service account for the SysKit Trace windows service. The alternative would be to put the certificate in the local machine certificate store and such a thing would be unsafe on shared machines since the certificate would be available to all users.
```
Comment (hbagarich) [inline installation-and-configuration/office-365-connection-details.md]: I would exclude this paragraph?

File: installation-and-configuration/office-365-connection-details.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+## SysKit Trace Azure AD Application
+When configuring SysKit Trace an Azure Active Directory Application will be created and pre-consented for the user to collect the data from the tenant.
+You can find the SysKit Trace application in the [Azure Portal](https://portal.azure.com/#blade/Microsoft_AAD_IAM/ActiveDirectoryMenuBlade/RegisteredApps) once SysKit Trace has been configured. 
+
+There are two ways to authenticate an Azure Active Directory Application with Azure AD when using the applications identity.
+1. application secrets
+2. certificates  
+
+SysKit Trace uses certificates because most of the workloads that are collected support it.
```
Comment (hbagarich) [inline installation-and-configuration/office-365-connection-details.md]: Maybe mention that certificates are more reliable?

Comment (hbagarich) [reply installation-and-configuration/office-365-connection-details.md]: And maybe link to documentation on it?

File: installation-and-configuration/office-365-connection-details.md (hunk 1 above)
Comment (vinkobedek) [reply installation-and-configuration/office-365-connection-details.md]: to technical or other reasons?

Comment (BrunoDrljanovcan) [reply installation-and-configuration/office-365-connection-details.md]: Or maybe just exclude part that is explaining alternatives

Comment (hbagarich) [reply installation-and-configuration/office-365-connection-details.md]: This part where we explain why we need a password every time and what are the alternatives. :)

Comment (vinkobedek) [reply installation-and-configuration/office-365-connection-details.md]: removed the text. better?

File: installation-and-configuration/office-365-connection-details.md (hunk 2 above)
Comment (vinkobedek) [reply installation-and-configuration/office-365-connection-details.md]: not sure what to put as a link. 
i found https://docs.microsoft.com/en-us/azure/active-directory/develop/active-directory-certificate-credentials
and https://docs.microsoft.com/en-us/azure/active-directory/develop/identity-platform-integration-checklist#security has the certificates mentioned, bit none is nice enough

Comment (vinkobedek) [reply installation-and-configuration/office-365-connection-details.md]: ok, pushed a new commit with the link. could be better, but should be acceptable

**Review by hbagarich — APPROVED** [review]

**Pull request merged**